### ISSUE TITLE
Allow hiding filters on dashboards

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#25249]
**Is your feature request related to a problem? Please describe.**
FullApp embedding does not give the option to hide filters similar to Signed Embedding and Public Sharing with `#hide_parameters=...`.

**Describe the solution you'd like**
Similar functionality to `#hide_parameters=...` on Signed/Public, but just for normal dashboards.
Alternatively something similar to https://www.metabase.com/docs/latest/embedding/full-app-embedding#action_buttons

**Describe alternatives you've considered**
Duplicating all questions and dashboard, and adjusting all questions. Time consuming without having #3575 and #9083

Alternatively, inject CSS via a reverse-proxy to hide a specific filter, but it's rather limited what is possible to do in pure CSS (and would need to be revised before every upgrade of Metabase to ensure it's still working)

**Additional context**
Related #17884

**Comment by jatorna** [comment]
@paoliniluis, do you have any updates on this feature?

**Comment by dragonsahead** [comment]
@jatorna nope, we're making many changes in how embedding works, but there's no news about this. What's the problem you're facing?

**Comment by jatorna** [comment]
@paoliniluis I have a dashboard with a lot of filters and they use the 40% of the screen with 5 rows. I would like to only show the filter when it is active, having the possibility to collapse or hide the filters. Qlik does something similar

**Comment by tcarriernc** [comment]
We're finding that with the permission structure we are using today one group per 'customer', this is not scaling. We got some good feedback from our reps to adjust this and we will be doing this to scale - so thank you for that! 

However when doing this the sandbox id 'customer id' would need to be a filter on each of our external customer dashboards. We feel that by showing this 'customer id' filter to the customer - it takes away some of the 'metabase magic' and shows a little too much of what happens in the background...decreasing trust a bit in the data. If customers don't see that and it just works as expected...only see what they're supposed to, our customers feel that they are not just a number and more that the analytics we provide are more tailored for them.

This feature would be an integral part of our journey to embed analytics into our company's application.

I can see the value in @jatorna 's request as well - our use case is more along the lines of if this sandbox id filter is always fixed (customer of course would not be able to change this filter or use it all), hide it.

Cheers,
Thomas
